### ISSUE TITLE
lens does not build with GHC 7.10.1

### ISSUE THREAD

**Issue by jstolarek** [ekmett/lens#534]
Building `lens` with GHC 7.10.1 results in:

```
src/Control/Lens/Internal/Exception.hs:197:10:
    Could not deduce (Typeable Handling)
      arising from the superclasses of an instance declaration
    from the context (Reifies s (SomeException -> Maybe a),
                      Typeable a,
                      Typeable m,
                      Typeable s)
      bound by the instance declaration
      at src/Control/Lens/Internal/Exception.hs:197:10-113
    In the instance declaration for ‘Exception (Handling a s m)’
```

**Comment by hvr** [comment]
@jstolarek is this `lens-4.8`? I've got `lens-4.8` working here with GHC 7.10.1...

**Comment by glguy** [comment]
Lens-4.8 (the current released version) builds on GHC 7.10.1.

It looks like lens-4.7 had a base < 5 version bound on it (which appears to be what you were trying from the look of the error message. I've lowered the bound to < 4.8 on hackage for len-4.7
